scripts/fix-script-order.sh

```
@@ -0,0 +1,47 @@
+for file in $HTML_FILES; do
+  # Check if command-data.js loads after command-palette.js
+  if grep -q "command-data.js" "$file" && grep -q "command-palette.js" "$file"; then
+    # Get line numbers
+    PALETTE_LINE=$(grep -n "command-palette.js" "$file" | head -1 | cut -d ":" -f 1)
+    DATA_LINE=$(grep -n "command-data.js" "$file" | head -1 | cut -d ":" -f 1)
+    
+    if [ "$PALETTE_LINE" -gt "$DATA_LINE" ]; then
+      echo "Fixing script order in $file..."
+      
+      # Remove the command-palette.js script line
+      PALETTE_SCRIPT_LINE=$(grep -n "command-palette.js" "$file" | head -1 | cut -d ":" -f 2-)
```

Shellcheck (0.10.0)

[warning] PALETTE_SCRIPT_LINE appears unused. Verify use (or export if used externally).

(SC2034)